DUI3-DX/Converters/Autocad/Speckle.Converters.AutocadShared/AutocadConverterToHost.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using Autodesk.AutoCAD.DatabaseServices;
+using Speckle.Autofac.DependencyInjection;
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Core.Models;
+
+namespace Speckle.Converters.Autocad;
+
+public class AutocadConverterToHost : ISpeckleConverterToHost
+{
+  private readonly IFactory<string, ISpeckleObjectToHostConversion> _toHost;
+  private readonly IHostToSpeckleUnitConverter<UnitsValue> _unitConverter;
+
+  public AutocadConverterToHost(
+    IFactory<string, ISpeckleObjectToHostConversion> toHost,
+    IHostToSpeckleUnitConverter<UnitsValue> unitConverter
+  )
+  {
+    _toHost = toHost;
+    _unitConverter = unitConverter;
+  }
+
+  public object Convert(Base target)
+  {
+    if (target is not Base speckleObject)
+    {
+      throw new NotSupportedException(
+        $"Conversion of {target.GetType().Name} to Host is not supported. Only objects that inherit from Base are."
+      );
+    }
```

Comment:
How would the `target` input not be a `Base`? It already expects it to be `Base` so this check seems redundant.